CactusTutorial.ipynb

```diff
@@ -180,40 +214,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!./simfactory/bin/sim create-submit helloworld \\\n",
-    "    --parfile arrangements/CactusExamples/HelloWorld/par/HelloWorld.par --walltime 0:5:0"
+    "%%bash\n",
+    "./simfactory/bin/sim create-run helloworld \\\n",
+    "    --parfile arrangements/CactusExamples/HelloWorld/par/HelloWorld.par"
```

Comment:
This does not work for me without specifying --ppn-used=1.

Reply:
This was discussed offline. This line (and the submit one later) produce a warning of the form
```
Warning: Too many threads per process specified: specified num-threads=6 (ppn-used is 12)
Warning: Total number of threads and number of threads per process are inconsistent: procs=1, num-threads=6 (procs*num-smt must be an integer multiple of num-threads)
Warning: Total number of threads and number of cores per node are inconsistent: procs=1, ppn-used=12 (procs must be an integer multiple of ppn-used)
```
which is non-fatal and happens whenever one requests less than a node's worth of cores from simfactory. There is nothing to fix here as the same warning will appear on clusters as well when running on less than one node's worth of cores.